docs/modules/operation/pages/provisioning/detectors/WmiDetector.adoc

```diff
@@ -10,46 +10,44 @@ By default, it expects to make a WMI query to the `Win32_ComputerSystem` class,
+| Parameter   | Default              | Description
+| username    | Administrator        | The username to use when authenticating against the WMI host.
+| password    | none                 | The password to use when authenticating against the WMI host.
+| domain      | WORKGROUP            | The Windows domain to use when authenticating against the WMI host.
+| wmiClass    | Win32_ComputerSystem | The class of instances within WMI. An example would be the `Win32_ComputerSystem` class. +
+                                        This class contains objects representing properties and methods available.
+| wmiObject   | Status               | A WMI object is technically a member of a WMI property set. +
+                                        Using the previous example of `Win32_ComputerSystem`, this class contains a variety of properties that we can look at and poll. +
+                                        The default WMI service uses the `Status` property to determine if the system is running and if WMI is available.
+| wmiWqlStr   | NOTSET               | Microsoft WMI also implements an `ExecQuery` method which is more common in Windows scripting but less common in OpenNMS polling and monitoring. +
+                                        WQL is a simple, SQL-like, syntax for querying WMI classes, properties, and instances. +
+                                        Here's an example of a WQL query that you could use to discover whether a Windows service is running: +
+                                        `Select State From Win32_Service Where Name='Server'`
+| compVal     | OK                   | The value to use when performing a comparison against the wmiObject.
+| compOp      | EQ                   | The compare operation is how you tell the OpenNMS WMI poller plugins how to verify the nature (up, critical) of a specific property within WMI. +
+                                        In the event of unequal comparisons, the system will always have the WMI value (as retrieved from the target) on the left. +
+                                        Available compare operations are: +
+                                        * *EQ* (equals) +
+                                        * *NEQ* (not equals) +
+                                        * *GT* (greater than) +
+                                        * *LT* (less than) +
+                                        * *NOOP* (no operation) +
+                                        The NOOP operation is a special case in which the Manager will always return a result code of "OK". +
+                                        This is typically used in scenarios where the existence of a class or property is more interesting than the contents of the property. +
+                                        We use this in the collector to check and ensure that WMI is available before attempting to collect data.
```

Comment:
```suggestion
                                        This can be used to ensure that WMI data is available before adding a collectd service based on the `WmiCollector` class.
```